article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -318,6 +318,64 @@ object DataModelV3 {
 
 object DotcomponentsDataModel {
   val VERSION = 2
```

Comment:
what are these server side changes ? 🤔

Reply:
It looks like a rebase has added in some file changes?